index.html

```diff
@@ -61,10 +62,198 @@ <h3>LET’S CONNECT</h3>
                 </ul>
 
             </div>
+                  <!-- start of work section -->
+
+<div class="work-container">
+    <div class="img-snapshot1">
+    <img src="images/Snapshoot1.png" alt="">
+    </div>
+    <div class="tonic1">
+        <h2>Tonic</h2>
+    </div>
+    <div class="frame">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+</div>
+    <button type="button" class="button-1">See Project</button>
+</div>
+    <!-- card2 -->
+             
+<section class="work-container1">
+    <div class="img-snapshot2">
+    <img src="images/Snapshoot2.png" alt="">
+    </div>
+    <div class="multipost">
+        <h2>Multi-Post
+            Stories</h2>
+    </div>
+    <div class="frame-1">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-1">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-1">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+
+</div>
+    <button type="button" class="button-2">See Project</button>
+
+ </section>
+    <!-- card3 -->
+             
+<section class="work-container2">
+    <div class="img-snapshot3">
+    <img src="images/Snapshoot3.png" alt="">
+    </div>
+    <div class="tonic2">
+        <h2>Tonic</h2>
+    </div>
+    <div class="frame-2">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-2">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-2">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+</div>
+  
+    <button type="button" class="button-3">See Project</button>
+
+</section>
+    <!-- card4 -->
+             
+<section class="work-container3">
+    <div class="img-snapshot4">
```

Comment:
- [ ] Following the [DRY principle](https://dev.to/richardwynn/dry-principle-in-100-seconds-2l8c) which is a good practice to include in your code, consider having one class for the "work-container" instead of having multiple classes (work-container1 - work-container2 - ...etc) for each card.